test/Python/numpy_conversion.swift

```diff
@@ -52,6 +52,15 @@ NumpyConversionTests.test("array-conversion") {
 
   let numpyArray2D = np.ones([2, 3])
   expectNil(Array<Float>(numpyArray: numpyArray2D))
+
+  let numpyArrayStrided = np.array([[1, 2], [1, 2]], dtype: np.int32)[
```

Comment:
Put the `[` on the next line.

Reply:
it thinks that `let numpyArrayStrided = np.array([[1, 2], [1, 2]], dtype: np.int32)` is over, and that `[Python.slice(Python.None), 1]` is a new expression if I do this.

Reply:
Ah, that makes sense.